packages/flutter/lib/src/gestures/converter.dart

```diff
@@ -104,90 +93,44 @@ class PointerEventConverter {
           );
```

Comment:
(github doesn't let me comment on the right line, this comment is meant for line 64):

Does it still make sense that this returns an iterable? Looks like it's a 1:1 mapping now?